packages/grid/_modules_/grid/components/tools/FilterForm.tsx

```diff
@@ -0,0 +1,190 @@
+  );
+
+  const changeLinkOperator = React.useCallback(
+    (event: React.ChangeEvent<{ value: unknown }>) => {
+      const linkOperator =
+        (event.target.value as string) === LinkOperator.And.toString()
+          ? LinkOperator.And
+          : LinkOperator.Or;
+      applyMultiFilterOperatorChanges(linkOperator);
+    },
+    [applyMultiFilterOperatorChanges],
+  );
+
+  const handleDeleteFilter = React.useCallback(() => {
+    deleteFilter(item);
+  }, [deleteFilter, item]);
+
+  return (
+    <div className={classes.root}>
+      <FormControl
+        className={classes.linkOperatorSelect}
+        style={{ width: 60, visibility: showMultiFilterOperators ? 'visible' : 'hidden' }}
+      >
+        <InputLabel id="columns-filter-operator-select-label">Operators</InputLabel>
+        <Select
+          labelId="columns-filter-operator-select-label"
+          id="columns-filter-operator-select"
+          value={multiFilterOperator}
+          onOpen={onSelectOpen}
+          onChange={changeLinkOperator}
+          disabled={!!disableMultiFilterOperator}
+          native
+        >
+          <option key={LinkOperator.And.toString()} value={LinkOperator.And.toString()}>
+            And
+          </option>
+          <option key={LinkOperator.Or.toString()} value={LinkOperator.Or.toString()}>
+            Or
+          </option>
+        </Select>
+      </FormControl>
+      <FormControl className={classes.columnSelect}>
+        <InputLabel id="columns-filter-select-label">Columns</InputLabel>
+        <Select
+          labelId="columns-filter-select-label"
+          id="columns-filter-select"
+          value={item.columnField || ''}
+          onChange={changeColumn}
+          onOpen={onSelectOpen}
+          native
+        >
+          {filterableColumns.map((col) => (
+            <option key={col.field} value={col.field}>
+              {col.headerName || col.field}
+            </option>
+          ))}
+        </Select>
+      </FormControl>
+      <FormControl className={classes.operatorSelect}>
+        <InputLabel id="columns-operators-select-label">Operators</InputLabel>
+        <Select
+          labelId="columns-operators-select-label"
+          id="columns-operators-select"
+          value={item.operatorValue}
+          onOpen={onSelectOpen}
+          onChange={changeOperator}
+          native
+        >
+          {currentColumn?.filterOperators?.map((operator) => (
+            <option key={operator.value} value={operator.value}>
+              {operator.label}
+            </option>
+          ))}
+        </Select>
+      </FormControl>
+      <FormControl className={classes.FilterValueInput}>
+        {currentOperator &&
+          React.createElement(currentOperator.InputComponent, {
+            item,
+            applyValue: applyFilterChanges,
+          })}
+      </FormControl>
+      <FormControl>
+        <IconButton
+          color="primary"
+          aria-label="Delete"
```

Comment:
No need to capitalize, but we can use the same translation for the title:

```suggestion
          aria-label="Delete"
          title="Delete"
```